package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "aws-cognito-wrapper",
+  "version": "1.0.0",
+  "description": "This is a wrapper for the amazon-cognito-identity-js project.",
+  "main": "src/index.js",
```

Comment:
This should probably point to your dist/ folder generated file if you want your module to workout the upstream project requiring a webpack configuration